client/src/services/user.ts

```diff
@@ -222,3 +224,27 @@ export type ProfileInfo = {
   stageInterviewFeedback?: StageInterviewDetailedFeedback[];
   discord: Discord | null;
 } & ProfileDto;
+
+export type ProfileMainCardData = {
+  location: Location | null;
+  name: string;
+  githubId: string | null;
+  publicCvUrl: string | null;
+};
+
+export enum ContactsKeys {
+  epamEmail = 'epamEmail',
+  email = 'email',
+  telegram = 'telegram',
+  phone = 'phone',
+  skype = 'skype',
+  notes = 'notes',
+  linkedIn = 'linkedIn',
+}
```

Comment:
I have 2 suggestions for little improvement:
1. As a general rule enum members are named in PascalCase (`epamEmail` => `EpamEmail` and so on)
2. If your enum doesn't contain computed members you can use `export const enum ContactsKeys` - it compiles into just set of values instead of  complex object https://www.typescriptlang.org/docs/handbook/enums.html#const-enums

Reply:
1. Noted, thanks. 
2. Slipped my mind, thanks for reminder.